atmos_model.F90

```diff
@@ -1764,7 +1764,6 @@ subroutine assign_importdata(rc)
                   nb = Atm_block%blkno(i,j)
                   ix = Atm_block%ixp(i,j)
 
-                  IPD_Data(nb)%Sfcprop%fice(ix)          = zero
```

Comment:
The line may set lake ice to be zero, can we confirm the impact of removing this line on lake ice?

Reply:
Yes, I have confirmed in my own experiments that ice fractionas over lakes are maintained after this fix.
The motivation that lead to finding this bug was to understand why ice concentration over lakes vanished in the coupled runs while the initial condition had non zero values